assets/js/src/core/modules/asset/editor/shared-tab-manager/tabs/tags/components/assigned-tags/assigned-tags.tsx

```diff
@@ -11,20 +11,41 @@
 *  @license    https://github.com/pimcore/studio-ui-bundle/blob/1.x/LICENSE.md POCL and PCL
 */
 
-import React from 'react'
-import { type Tag } from '@Pimcore/modules/asset/editor/shared-tab-manager/tabs/tags/tags-api-slice.gen'
+import React, { useState } from 'react'
+import {
+  type Tag,
+  useUnassignTagFromElementMutation
+} from '@Pimcore/modules/asset/editor/shared-tab-manager/tabs/tags/tags-api-slice.gen'
 import { createColumnHelper } from '@tanstack/react-table'
 import { useTranslation } from 'react-i18next'
 import { Icon } from '@Pimcore/components/icon/icon'
-import { Button } from 'antd'
+import { Button, Result } from 'antd'
 import { Grid } from '@Pimcore/components/grid/grid'
+import { useStyle } from './assigned-tags.styles'
+import { useGlobalAssetContext } from '@Pimcore/modules/asset/hooks/use-global-asset-context'
 
 type TagWithActions = Tag & {
   actions: React.ReactNode
 }
 
 export const AssignedTagsTable = ({ tags, isLoading }: { tags: Tag[], isLoading: boolean }): React.JSX.Element => {
   const { t } = useTranslation()
+  const [loadingRows, setLoadingRows] = useState({})
+  const { context } = useGlobalAssetContext()
```

Comment:
Can we use the normal Asset Context in that case? 

The global context is set a bit later in the render cycle, than the normal asset context. 
That would save us some extra renders. :)

Reply:
I used the `useGlobalAssetContext` to also get the type since we could reuse it for other element types ...

Reply:
But the useGlobal**Asset**Context is also only used for assets. 

For sure we will have to add an additional context for other element types.
But that's something that I would see under a separate/new context.

Reply:
And something we should face in the moment when we start with the object area.